content/en/docs/setup/install/multicluster/virtual-machine/index.md

```diff
@@ -0,0 +1,156 @@
+---
+title: Installing Istio for virtual machine connectivity
+description: Install an Istio mesh such that a virtual machine can connect to it and configure the virtual machine.
+weight: 3
+keywords:
+- kubernetes
+- virtual-machine
+- gateways
+- vms
+---
+
+Follow this guide to install an Istio control plane
+[multicluster deployment](/docs/ops/deployment/deployment-models/#multiple-clusters) and connect your first virtual machine.
+
+## Prerequisites
+
+1. [Download the Istio release](/docs/setup/getting-started/#download)
+1. Perform any necessary [platform-specific setup](/docs/setup/platform-setup/)
+1. Check the requirements [for Pods and Services](/docs/ops/deployment/requirements/)
+1. Virtual machines must have IP connectivity to the ingress gateway in the connecting mesh, and optionally every pod in the mesh via L3 networking if enhanced performance is desired.
+
+## Installing the Istio control plane
+
+1. Set the name of your cluster as an environment variable. Please be aware
+    that cluster names may be autogenerated by your Kubernetes cloud provider.
+    Therefore, the cluster name you specify during creation of your cluster may
+    be different than the name of the cluster expected here. This will be evident
+    by invalid x509 certificate errors in the Istio logs after completing
+    installation.
+
+    {{< text bash >}}
+    $ CLUSTER_NAME="<the name of your cluster>"
+    {{< /text >}}
+
+1. Create a working directory for files generated in this documentation.
+ 
+    {{< text bash >}}
+    $ mkdir -p "${HOME}"/"${CLUSTER_NAME}"
+    {{< /text >}}
+
+1. Execute the following commands on each cluster in the mesh where a virtual
+    machine connection should be established. See
+    [Certificate Authority (CA) certificates](/docs/tasks/security/cert-management/plugin-ca-cert/)
+    for more details on configuring an external CA.
+
+    {{< text bash >}}
+    $ make -C sample/certs NAME="${CLUSTER_NAME}" NAMESPACE="vm" "${CLUSTER_NAME}"-certs-wl
```

Comment:
Please make it very clear this is for a sample of how certs can be generated - not the only or recommended method.

Reply:
will add a callout, thanks.

Reply:
we set `NAME` but as far as I can tell this is not used in the makefile?

Reply:
@irisdingbj can you offer more context on the NAME variable.

Reply:
@irisdingbj can you offer more context on the NAME variable.

Reply:
we can directly `make -C sample/certs NAMESPACE="vm" "${CLUSTER_NAME}"-certs-wl`

Reply:
I was asking why the NAME variable exists at all. After reading the README.md in the directory, I see it is used to uniquely identify clusters n the location field of the cert.